household-esignet-integration-impl/README.md

```diff
@@ -1,5 +1,54 @@
-# mock-esignet-integration-impl
+# HouseHold ID e-Signet plugin implementation
 
 ## About
+e-Signet integration with householdID view. Here the householdID DB view plays the role of ID repository. IDA implementation to identify users based on the provided ID number is part of the authenticator plugin.
 
-Implementation for all the interfaces defined in esignet-integration-api.
\ No newline at end of file
+KYC returned as part of userinfo response will only contain the subject claim. Subject claim will hold the household ID.
```

Comment:
We are giving out household ID directly in the subject claim ? this will not cause privacy issue ?

Reply:
It was the requirement from HHID team. We had suggested to give out PSUT -> hash of HHID & RelyingParty ID